vendor/setup_window.ps1

```diff
@@ -12,6 +54,15 @@ $bonjour_sdk = "$storageDir\bonjoursdksetup.exe"
 $python_remote = "https://www.python.org/ftp/python/2.7.10/python-2.7.10.amd64.msi"
 $python = "$storageDir\python.msi"
 
+Write-Host " "
+Write-Host "Downloading Node: $node_installer"
+$webclient.DownloadFile($node_installer_remote, $node_installer)
```

Comment:
Installing this could be bad as users could already have specific node versions installed.  We need to somehow ask the user for input.  I'll look into this in PowerShell